tests/prefetch_related/tests.py

```diff
@@ -626,6 +626,16 @@ def test_custom_qs(self):
         room = Room.objects.filter(main_room_of__isnull=False).prefetch_related(Prefetch('main_room_of', queryset=houses.filter(address='DoesNotExist'), to_attr='main_room_of_attr')).first()
         self.assertIsNone(room.main_room_of_attr)
 
+    def test_nested_prefetch_related_are_not_overwritten(self):
+        houses_2 = House.objects.prefetch_related(Prefetch('rooms'))
+        persons = Person.objects.prefetch_related(Prefetch('houses', queryset=houses_2))
+        houses = House.objects.prefetch_related(Prefetch('occupants', queryset=persons))
+        list(houses)
```

Comment:
I guess this line is to evaluate the queryset, but seems a bit odd since that would happen in the next line, wouldn't it?